src/test/java/org/jenkinsci/plugins/envinject/BuildCauseRetrieverTest.java

```diff
@@ -39,13 +40,14 @@
     @SuppressWarnings("deprecation")
     @Test
     public void shouldWriteInfoAboutManualBuildCause() throws Exception {
-        Cause cause = Cause.UserCause.class.newInstance();
+    	UserIdCause cause = Cause.UserIdCause.class.newInstance();
```

Comment:
I just wonder how it may work, because you don't specify the username when you trigger the build... If the check below works, then something is wrong with the matcher